.github/pull_request_template.md

```diff
@@ -1,12 +1,49 @@
+# Thank you for submitting a pull request to newrelic/developer-website
+
```

Comment:
Could we _remove_ this from the template rather than adding it?

I've noticed that many folks don't bother deleting this line (and some of the other boilerplate messaging) which adds to the visual clutter in PR descriptions.

Reply:
Sure.